src/resources/locales/en.json

```diff
@@ -271,15 +271,15 @@
             "title": "Duration"
         },
         "expired": "Loan expired",
-        "expiredLcp": "This book can't be read because the LCP licence has expired.",
+        "expiredLcp": "Expired LCP license.",
```

Comment:
better -> "The LCP license has expired". (to be even with returnedLCP and revokedLCP)

Reply:
indeed!